### ISSUE TITLE
Error: Cannot read properties of undefined (reading 'testsuite')

### ISSUE THREAD

**Issue by lorenooliveira** [ctrf-io/junit-to-ctrf#3]
I was giving `junit-to-ctrf` a try but seems like it cannot parse the test report. 

TLDR:

```bash
$ npx junit-to-ctrf TEST-com.example.demo.DemoApplicationTests.xml -o test.json

Reading JUnit report file: TEST-com.example.demo.DemoApplicationTests.xml
Error: Cannot read properties of undefined (reading 'testsuite')
```

Steps to reproduce:

1. Create the simplest possible project in https://start.spring.io/
    - Maven project
    - Java language
    - All defaults to Project Metadata
    - War packaging
    - Java 21
2. Unzip project locally
3. mvn clean test
4. cd target/surefire-reports
5. npx junit-to-ctrf TEST-com.example.demo.DemoApplicationTests.xml -o test.json

Notable environment details:
- node --version   -> v20.18.1
- npm --version   -> 10.8.2
- npx --version    -> 10.8.2
- java version    -> corretto 22
- ubuntu 24

**Comment by Ma11hewThomas** [comment]
I think this might be a surefire report? @sahilwingify is building a [surefire-to-ctrf](https://github.com/orgs/ctrf-io/discussions/18) converter which should be available soon

**Comment by lorenooliveira** [comment]
Good to know. I'm monitoring the other discussion.

Just out of curiosity, if not surefire's/failsafe's test reports, what is the junit test report expected by this package?

**Comment by Ma11hewThomas** [comment]
fixed in #6, published 0.0.5

**Comment by lorenooliveira** [comment]
Just confirming that it looks good now 👍
